### ISSUE TITLE
Stop sequence validation error for `gpt-4-vision-preview`

### ISSUE THREAD

**Issue by cameronk** [promptfoo/promptfoo#366]
Calls to `openai:chat:gpt-4-vision-preview` fail due to a validation error with the `stop` sequence parameter.

Example config:
```
prompts:
  - "Write a tweet about {{topic}}"
  - "Write a very concise, funny tweet about {{topic}}"

providers: [openai:chat:gpt-4-vision-preview]

tests:
  - vars:
      topic: bananas
```

Result:
<img width="1157" alt="image" src="https://github.com/promptfoo/promptfoo/assets/424892/45d2826e-bcb5-4916-94b8-6bafe84f8cc3">

I fixed this in my config by just setting the stop sequence value like so:
```
providers: 
  - id: openai:chat:gpt-4-vision-preview
    config:
      stop: 
        - <|endoftext|>
```

**Comment by typpo** [comment]
Thanks @cameronk.  Just pushed a fix that will be included in the next release.  It looks like `gpt-4-vision-preview` handles undefined `stop` differently from other chat models.

**Comment by typpo** [comment]
Fix is in 0.34.0
